splunk_add_on_ucc_framework/commands/rest_builder/endpoint/single_model.py

```diff
@@ -161,6 +162,11 @@ def oauth_client_credentials_call(self):
             ).body.read().decode("utf-8")
         )["entry"][0]["content"]
 
+        if "access_token" not in data:
+            if "error" in data:
+                data = data["error"]
+            raise InternalException("Error while trying to obtain OAuth token: %s" % data)
```

Comment:
Does our UCC framework (UI) correctly parse and show the raised "InternalException"? As far as I know, I think we have to raise RestError from `splunktaucclib.rest_handler.error`.